EvtWeightTools/src/Data_to_MC_CorrectionInterface_1l_2tau_trigger.cc

```diff
@@ -5,33 +5,14 @@
 #include "TallinnNtupleProducer/CommonTools/interface/leptonTypes.h"                            // getLeptonType()
 #include "TallinnNtupleProducer/CommonTools/interface/sysUncertOptions.h"                       // TriggerSFsys
 #include "TallinnNtupleProducer/EvtWeightTools/interface/data_to_MC_corrections_auxFunctions.h" // aux::
-#include "TallinnNtupleProducer/Objects/interface/RecoHadTau.h"                                 // RecoHadTau
-#include "TallinnNtupleProducer/Objects/interface/RecoLepton.h"                                 // RecoLepton
 #include "TallinnNtupleProducer/CommonTools/interface/hadTauDefinitions.h"
 
 #include <assert.h>                                                                             // assert()
+#include <TMath.h>                                                                       
+// std::fabs()
```

Comment:
`std::fabs()` is available in the STL (`<cmath>`), which should already be included by the other headers, so not needed.

Reply:
done